src/Tribe/Admin/Notice/Php_Version.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Shows an admin notice for Php_Version
+ */
+class Tribe__Admin__Notice__Php_Version {
+
+	public function hook() {
+
+		// display the PHP version notice
+		tribe_notice(
+			'tribe-events-php-notice',
```

Comment:
Since we are in common, and `tribe_notice` it will add a `tribe` namespace. So we can call it `php-deprecated`.